javascript-timers/interval.js

```diff
@@ -0,0 +1,23 @@
+var $title = document.querySelector('.countdown-display');
+
+function changeTitle(title1, title2, title3, title4) {
+  if ($title.textContent === '4') {
+    $title.textContent = title1;
+  } else if ($title.textContent === title1) {
+    $title.textContent = title2;
+  } else if ($title.textContent === title2) {
+    $title.textContent = title3;
+  } else if ($title.textContent === title3) {
+    $title.textContent = title4;
+    clearInterval(interval);
+    interval = null;
+  }
+}
```

Comment:
Instead of doing all these `else if` to handle each countdown, you should be writing this in a dynamic way to handle for all cases. That way you can have the number start from anywhere. Think about how you can use either the text content or a variable to use this to your advantage